cppcache/src/ExpiryTaskManager.cpp

```diff
@@ -58,7 +58,20 @@ long ExpiryTaskManager::scheduleExpiryTask(ACE_Event_Handler* handler,
 
   ACE_Time_Value expTimeValue(expTime);
   ACE_Time_Value intervalValue(interval);
-  LOGFINER("Scheduled expiration ... in %d seconds.", expTime);
+  return m_reactor->schedule_timer(handler, 0, expTimeValue, intervalValue);
+}
+
+long ExpiryTaskManager::scheduleExpiryTask(
```

Comment:
I assume this one is still in-progress?  The old method above still takes uint32_t

Reply:
Its used internal still. There is another ticket for all internal time.

Reply:
Okay, cool.